src/main/java/uk/gov/hmcts/reform/sscs/services/CaseLoaderService.java

```diff
@@ -231,7 +243,16 @@ private List<SscsCaseDetails> getCasesByCaseRefOrCaseIdWithInvalidReferenceError
         IdamTokens idamTokens
     ) {
         try {
-            return searchCcdCaseService.findListOfCasesByCaseRefOrCaseId(caseData, idamTokens);
+            if (caseLoaderUpdateCaseV2Enabled) {
```

Comment:
What's the reason for initiating an event here? we're not tracking the token from it. Also is this a change of behaviour from the previous lookup method which looked at ref or id?

Reply:
@banderous this is an Elastic search query, I'm replacing it with `startEvent` call to get the latest data as we don't have another API endpoint to get data from ccd db